### PR TITLE
feat : Added Malayalam translation #231

### LINKED ISSUE #231: Malayalam translation of title template, headings, and labels
**Summary**
Add Malayalam (`ml`) translations of title template, headings, labels, etc for locale code `ml`.

**Potential Contributors**
See [Contributing a Translation](https://github.com/cicirello/user-statistician#contributing-a-translation) section of the README for step-by-step instructions.

Please don't edit the README with the new locale code, despite the Pull Request template requesting that documentation is updated. Changes to the README automatically go live in the GitHub Marketplace listing for the action. I'll take care of updating README with the new locale code in the list of locales when I create the release that contains it to ensure that documentation is consistent with latest release.

### PR DESCRIPTION
**Pull request by Sarthak027**
## Summary
Adding Malayalam translations

## Closing Issues
closes #231 

## Types of changes

- [x] I have read the [**CONTRIBUTING**](https://github.com/cicirello/.github/blob/main/CONTRIBUTING.md) document.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Improvements to existing code, such as refactoring or optimizations (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/locales/ml.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "titleTemplate": "{0}-ന്റെ GitHub പ്രവർത്തനം",
```
Comment (Viveksati5143) [inline src/locales/ml.json:2]: Is there some word for GitHub in Malayalam?

**Review by Viveksati5143 — CHANGES_REQUESTED** [review]

File: src/locales/ml.json (hunk 1 above)
Comment (cicirello) [reply src/locales/ml.json:2]: @Viveksati5143 thanks for reviewing.
@Sarthak027 thanks for confirming.

**Comment by cicirello** [comment]
Thanks for contributing @Sarthak027. Later this week I'll create a release that includes this.

**Comment by Sarthak027** [comment]
Sure @cicirello.

**Comment by cicirello** [comment]
@Sarthak027 I just released v1.21.0 that includes this translation (also available via major tag v1). 🚀 Thank you again for contributing.

**Pull request merged**